src/components/DonorOverview.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react'
+import './DonorRecords.css'
+import Form from 'react-bootstrap/Form'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import EditDonorForm from './EditDonorForm'
+import NavBarWrapper from '../helpers/NavBarWrapper'
+
+
+const DonorRecords = () => {
+  const currentDonor = {
+    name: 'Amy Lim',
+    email: 'amylim@gmail.com',
+    nric: 'S1239879D',
+    type: 'Recurring',
+    salutation: 'Miss',
+    phone: '91234567',
+    address: '123 Tampines West #11-01 Singapore 234567'
+  }
+  const [ donor, setDonor ] = useState(currentDonor)
+  const [ updatedDonor, setUpdatedDonor ] = useState(donor)
+  const [ editing, setEditing ] = useState(false)
+
+  const getUpdatedDonor = donorData => {
+    setUpdatedDonor(donorData)
+  }
+
+  const updateDonor = () => {
+    setEditing(false)
+    setDonor(updatedDonor)
+  }
+  const SaveButton = () => (
+    <button onClick={updateDonor} className="btn btn-primary">Save edits</button>
+  )
+
+  const CancelButton = () => (
+    <button onClick={()=>setEditing(false)} className="btn btn-outline-primary">Cancel</button>
+  )
+
+  const EditButton = () => (
+    <button onClick={()=>setEditing(true)} className="btn btn-outline-primary">Edit donor</button>
+  )
+
+  /*render*/
+  return(
+      <main>
+      <h1>Donor Records</h1>
+      <div className="breadcrumbs">
+        <p>Donor List  >  <strong>{donor.name}</strong></p>
+        { editing ? <div className="btn-toolbar mr-2"><SaveButton/><CancelButton/></div> : <EditButton/> }
```

Comment:
For this one, the save button and cancel will need some spacing here. let's add a small margin as a spacing between the buttons